unit1-wrap-up/activity-mind-map.md

```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
```

Comment:
Could we conform this to:

```suggestion
# Activity: Mind Map
```

? I'm really hoping for file names, lesson names, what's on the config, and what shows up on Learn, to all be aligned (so that we don't lose files as easily as we did in textbook-curriculum)

Reply:
incorporated.